django/db/backends/postgresql/schema.py

```diff
@@ -15,6 +15,8 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     sql_create_index = "CREATE INDEX %(name)s ON %(table)s%(using)s (%(columns)s)%(extra)s%(condition)s"
     sql_delete_index = "DROP INDEX IF EXISTS %(name)s"
 
+    sql_create_column_inline_fk = "REFERENCES %(to_table)s (%(to_column)s) DEFERRABLE INITIALLY DEFERRED"
+
```

Comment:
OK. I guess the code paragraphs here are organized according to the type of thing being created/deleted? This one's a gray area for me because, although the attribute name ends in `_fk` like the attribute below, this particular attribute relates to creating a FK constraint _while_ creating a column.

Not that it really matters. I'll delete the line.